postgres/codenamemap.yaml

```diff
@@ -54,12 +54,14 @@
 {{ debian_codename('wheezy', '9.1') }}
 {{ debian_codename('jessie', '9.4') }}
 {{ debian_codename('stretch', '9.6') }}
+{{ debian_codename('buster', '11') }}
 
 # `oscodename` grain has long distro name
 # if `lsb-release` package not installed
 {{ debian_codename('wheezy', '9.1', 'Debian GNU/Linux 7 (wheezy)') }}
 {{ debian_codename('jessie', '9.4', 'Debian GNU/Linux 8 (jessie)') }}
 {{ debian_codename('stretch', '9.6', 'Debian GNU/Linux 9 (stretch)') }}
+{{ debian_codename('buster', '11', 'Debian GNU/Linux 10 (buster)') }}
```

Comment:
shouldn't this be '10' ?

Reply:
No, that's a mistake I made as well :p this adds the PostgreSQL version that exists in the distro repo and doesn't refer to the distrio version at all.

Reply:
thanks for pionting this out.